src/tasks/AndroidAppBuilder/Templates/monodroid-coreclr.c

```diff
@@ -200,7 +200,7 @@ mono_droid_runtime_init (const char* executable)
 
     chdir (g_bundle_path);
 
-    struct host_runtime_contract host_contract = {
+    static struct host_runtime_contract host_contract = {
```

Comment:
How about elevating this outside of the method and making it `static` global? If I recall, inner global types are only initialized when the function is entered and have additional cost.